types/node/index.d.ts

```diff
@@ -118,12 +118,17 @@ interface NodeRequireFunction {
 }
 
 interface NodeRequire extends NodeRequireFunction {
-    resolve(id: string): string;
+    resolve: RequireResolve;
     cache: any;
     extensions: NodeExtensions;
     main: NodeModule | undefined;
 }
 
+interface RequireResolve {
+    (id: string, options?: { paths: string[]; }): string;
+    paths(request: string): string[];
```

Comment:
`require.resolve.path()` may return also `null` according to docu.

Reply:
Got it.